docs/technical-guide/developer/backend.md

```diff
@@ -4,14 +4,22 @@ title: 3.06. Backend Guide
 
 # Backend guide #
 
-This guide intends to explain the essential details of the backend
-application.
-
+This guide collects some basic information on the backend application.
 
 ## REPL ##
 
-In the devenv environment you can execute <code class="language-clojure">scripts/repl</code> to open a
-Clojure interactive shell ([REPL](https://codewith.mu/en/tutorials/1.0/repl)).
+_Note:_ When in development mode, the backend spins up a traditional nREPL socket on port 6064.
+If you are experimenting locally, you can connect to it using your Clojure editor or
+with `backend/scripts/nrepl`, which starts a [REPLy client](https://github.com/trptcolin/reply),
+[see here][1] for more information.
+
+[1]: /technical-guide/developer/devenv/#backend
+
+In the devenv environment you can execute `backend/scripts/repl` to open a
```

Comment:
devenv environment -> development environment

Reply:
Note  that "devenv" seems to be the convention in the documentation, with 36 hits vs 14 for "development environment". Fixing this should be the purpose of a different PR, IMO.